simphony_ui/liggghts_model.py

```diff
@@ -14,48 +15,184 @@ class LiggghtsModel(HasStrictTraits):
+    wall_cohesion_energy_density_flow = Float(0.0)
+
+    wall_pair_potentials = Enum('repulsion', 'cohesion')
+
+    restitution_coefficient_title = Str('Restitution coefficient')
+    friction_coefficient_title = Str('Friction coefficient')
+    cohesion_energy_density_title = Str('Cohesion energy density')
 
     traits_view = View(
         VGroup(
-            Item(name='timestep'),
-            Item(name='num_iterations', label='Number of iterations'),
-            '_',
-            Item(name='input_file'),
-            '_',
-            HGroup(
-                Item(name='boundary_condition_x', show_label=False),
-                Item(name='boundary_condition_y', show_label=False),
-                Item(name='boundary_condition_z', show_label=False),
-                label='Boundary conditions'
+            VGroup(
+                Item(name='timestep'),
+                Item(name='num_iterations', label='Number of iterations'),
+                '_',
+                Item(name='input_file'),
+                label='Computational method parameters',
+                show_border=True
             ),
-            '_',
-            Item(name='young_modulus'),
-            Item(name='poisson_ratio'),
-            Item(name='restitution_coefficient'),
-            Item(name='friction_coefficient'),
-            Item(name='cohesion_energy_density'),
-            HGroup(
-                Item(name='pair_potentials_1', show_label=False),
-                Item(name='pair_potentials_2', show_label=False),
-                label='Pair potentials'
+            VGroup(
+                HGroup(
+                    Item(name='x_wall'),
+                    Item(name='y_wall'),
+                    Item(name='z_wall'),
+                ),
+                '_',
+                Item(name='flow_particles_fixed'),
+                Item(name='wall_particles_fixed'),
+                label='Boundary conditions',
+                show_border=True
             ),
+            VGroup(
+                VGroup(
+                    Item(
+                        name='flow_young_modulus',
+                        label='Young\'s modulus'
+                    ),
+                    Item(
+                        name='flow_poisson_ratio',
+                        label='Poisson\'s ratio'
+                    ),
+                    '_',
+                    Item(
+                        name='restitution_coefficient_title',
+                        style='readonly',
+                        show_label=False,
+                    ),
+                    Item(
+                        name='flow_restitution_coefficient_flow',
+                        label='with flow particles'
+                    ),
+                    Item(
+                        name='flow_restitution_coefficient_wall',
+                        label='with wall particles'
+                    ),
+                    '_',
+                    Item(
```

Comment:
Actually, looking at the way things are structured here, it may be better to remove the `'_'` dividers and instead use `VGroup`s with and appropriate `label` value as the title.